### PR TITLE
Shorten official format names

### PR DESCRIPTION
**Pull request by ry42alt**
re: [Original Discord message](https://discord.com/channels/630837856075513856/630973468409856020/1174623272348946523)

As is, the names for official cartridge formats are kind of long - so long that some of them get cut off!
image

The main way I aim to fix this is to completely remove the year from BSS/VGC format names, and shorten "Battle Stadium Singles" to "BSS".

As DaWoblefet says [**here**](https://discord.com/channels/630837856075513856/630973468409856020/1174696102755307560), cartridge formats were originally named this way because they used to function this way - the format would be the same for the entire year, thus making the year necessary in distinguishing between formats. However, now that Sword/Shield and Scarlet/Violet have roatating formats, the year has become less and less meaningful.

Shortening to BSS is less important, but just as meaningful. Most players know what BSS is by now, and those that don't will have had to ask anyways ("whats BSS" vs "whats battle stadium singles").

Preview of what both of these changes would look like: 
image

TLDR: Specifying the generation and regulation letter are more than enough to distinguish between different BSS/VGC formats without including the year.

--i totally understand if importer.ts can't/shouldn't be changed, since that has to do with something more than just showdown

### CODE REVIEW

**Comment by DaWoblefet** [comment]
We already started to adopt a new standard for VGC/BSS naming convention:

VGC [year] Reg [regulation]
BSS Reg [regulation]

We're not dropping the year from VGC formats. We didn't update previous format names to not inconvenience folks by moving all their teams to a different format, affecting the teambuilder. Regulation E will go away at the end of the month.
